src/rpc/net.cpp

```diff
@@ -272,6 +272,49 @@ static RPCHelpMan getpeerinfo()
     };
 }
 
+static RPCHelpMan addwhitelist()
+{
+    return RPCHelpMan{"addwhitelist",
+                "\nAdd permission flags to the peers using the given IP address (e.g. 1.2.3.4) or\n"
+                "CIDR-notated network (e.g. 1.2.3.0/24). Uses the same permissions as -whitebind.\n"
+                "Additional flags in and out control whether permissions apply to incoming connections and/or outgoing (default: incoming only).\n",
```

Comment:
Perhaps also useful to include this in the example?
```suggestion
                "Additional flags \"in\" and \"out\" control whether permissions apply to incoming and/or outgoing connections (default: incoming only).\n",
```